src/app/ubs/ubs-admin/components/shared/components/tariff-deactivate-confirmation-pop-up/tariff-deactivate-confirmation-pop-up.component.ts

```diff
@@ -26,6 +33,13 @@ export class TariffDeactivateConfirmationPopUpComponent implements OnInit {
   isRestore: boolean;
   isDeactivate: boolean;
   isDeactivatePopup = true;
+  lang = false;
```

Comment:
After added new function delete this.lang variable

Reply:
Thank you! Change code according to recommendations